prow/pubsub/subscriber/subscriber.go

```diff
@@ -238,6 +238,22 @@ func (s *Subscriber) handleProwJob(l *logrus.Entry, jh jobHandler, msg messageIn
 		return fmt.Errorf("failed getting prowjob spec") // This should not happen
 	}
 
+	// deny job that runs on not allowed cluster
+	var clusterIsAllowed bool
+	for _, allowedCluster := range allowedClusters {
+		if allowedCluster == "*" || allowedCluster == prowJobSpec.Cluster {
+			clusterIsAllowed = true
+			break
+		}
+	}
+	if !clusterIsAllowed {
+		err := fmt.Errorf("cluster %s is not allowed", prowJobSpec.Cluster)
```

Comment:
This isn't very actionable. Please include some brief instructions for adding the necessary config.

Reply:
done